charts/warpstream-agent/values.yaml

```diff
@@ -79,8 +79,10 @@ resources:
   requests:
     cpu: 2
     memory: 8Gi
+    ephemeral-storage: "100Mi"
```

Comment:
let's put a fat comment here to make sure people don't think we are storing data on disk?

Reply:
+1